### DISCUSSION TITLE
外部リンク記号の提供

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#240] [要望・アイデア・不具合]
## アイデア

記事を書く際にリンクを挿入する形式として、四角形と右上向き矢印を組み合わせた記号にリンクを埋め込むものがあります。

<img src="https://qiita-user-contents.imgix.net/https%3A%2F%2Fqiita-image-store.s3.ap-northeast-1.amazonaws.com%2F0%2F590705%2F5bbe92ea-c85b-2915-550d-1fff420cfe50.png?ixlib=rb-4.0.0&auto=format&gif-q=60&q=75&w=1400&fit=max&s=e208cde190a6ab9d6e2c10a67ade0f10" alt="External Link Symbol in Qiita site page" width=55%>

多くのサイトではこのような記号を利用する場合、CSS で作成するようです。Qiita に投稿される記事内では CSS 等による作成は出来ないため、次のような記事を作成しながらいくつかの方策を考えました :muscle:

> [Qiita 記事に外部リンク記号を挿入したい - Qiita](https://qiita.com/Yarakashi_Kikohshi/items/be5220d849c680a5a171)

実際、どれくらいの需要や利用価値があるか分かりませんが、外部リンク記号を簡単に挿入する方法が提供されていても良いかもしれません :eyes:

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見・検証記事をいただきありがとうございます🙏
初めて頂いたご意見かもしれません。今まで具体的に検討をしたことがないので調査してみます！
ただ、直近の開発優先度などから考えると、修正は先になってしまいそうです🙇
気長にお待ちいただけると幸いです🙇💦💦

**Reply by yarakos95** [reply]
返信ありがとうございます。

私としても、このアイデアが優先度の高いものだと考えてはいません。記号としてあったら良いなぁと思い提案させていただきました。

実現される日を楽しみにしています :+1:
